example/pubspec.lock

```diff
@@ -0,0 +1,89 @@
+# Generated by pub
```

Comment:
shouldn't commit this, right?

Reply:
Why not; this example is an app?

Reply:
Because it'll have random deltas as we work on it and it adds no benefit to the end user

Reply:
I think the example will very rarely change, so I don't think the deltas will be a big deal. Any example should reflect best practices for an app. However, for me the question is if we really want to maintain examples here at all (as opposed to a dedicated samples repo). I'm in favor of keeping this repo fairly pristine, showing the Dockerfile for this in the README that we'll publish to [official docs](https://github.com/docker-library/docs) under `dart`, but linking to a sample repo that we maintain elsewhere.